Sever config docs: Deploy security agents (#19236)
fleetdm/fleet: docs/Configuration/fleet-server-configuration.md

    sts_assume_role_arn: arn:aws:iam::1234567890:role/ses-role
  ```

#### S3

##### s3_bucket

> Name of the S3 bucket to use to store software and file carves.

Reviewer: ```suggestion
Name of the S3 bucket for storing software and file carves.
```
